structs.go

```diff
@@ -238,6 +238,9 @@ type Channel struct {
 
 	// A list of permission overwrites present for the channel.
 	PermissionOverwrites []*PermissionOverwrite `json:"permission_overwrites"`
+	
```

Comment:
This extra tab breaks the build :|

Reply:
Should be resolved and amended waiting for new build to complete